### PR TITLE
Darken shade of v btn hover

### PR DESCRIPTION
**Pull request by timothyjohn2015**
In https://trello.com/c/TjFr6i1c/120-design-todos

this is to address:

"Cancel in modals - hover should be a shade darker grey."

Changes this:
Screen Shot 2020-09-15 at 3 47 15 PM

To this:
Screen Shot 2020-09-15 at 3 47 24 PM

### CODE REVIEW

**Comment by timothyjohn2015** [comment]
Hold off on merging until we get signoff from Andrew on shade change, other feedback welcome though

**Review by sandemchad — APPROVED** [review]
👍 LGTM nice work

**Pull request merged**
